### ISSUE TITLE
The layout icons dont show the header and footer

### ISSUE THREAD

**Issue by docwilmot** [backdrop-contrib/radix_layouts#2]
Just a note, when I looked at the layout Icons, I expected the Layout edit page to have exactly those regions; seems that all your Layouts have a "header" and "footer" region in addition to the regions depicted in the icons. 
I assume thats by design, but maybe a note in the README?
I actually wonder if the header and footer regions are even necessary in your Layouts.

**Comment by shadcn** [comment]
Yep. Each layout has a header and footer that are not shown in the icons. I need to update the icons to reflect that.

The header and footer regions are there to let you have different layouts on different pages but at the same time it helps to keep logo, main navigation and footer in the same place.

On pages where you don't need header or footer, you can just remove the block from the region.

**Comment by docwilmot** [comment]
Pushed a new branch with updated icons. Merge when youre happy.

**Comment by shadcn** [comment]
Can you send it as PR? Thanks.

**Comment by docwilmot** [comment]
After all of this, seems all preview icons should be named "preview.png".
